setup.py

```diff
@@ -1,12 +1,35 @@
 #!/usr/bin/env python
 
-from distutils.core import setup
-
-setup(name='BeatCop',
-      version='1.0',
-      description='Cluster singleton manager',
-      long_description=open('README.md').read(),
-      url='https://github.com/Luluvise/BeatCop',
-      requires=['redis(>=2.9.1)'],
-      scripts=['beatcop.py'],
+from __future__ import print_function
+
+from setuptools import setup
+
+
+long_description = '...'
+for _fln in ('README.rst', 'README.md'):
+    try:
+        with open(_fln) as fo:
+            long_description = fo.read()
+    except Exception as exc:
+        pass
+    else:
+        break
+
+
+setup_kwargs = dict(
+    name='BeatCop',
+    version='1.0',
+    description='Cluster singleton manager',
+    long_description=open('README.md').read(),
```

Comment:
It's a different black magic:

First problem was that the packag isn't on pypi.

The second problem I ran into was that installing this as package threw an exception about not finding 'README.md'. Later I solved that problem by adding `MANIFEST.in`.

Another problem was that pypi supposedly only supports restructuredtext `rst`, not markdown `md`, so I added a `register.py` with a bit of support in the `setup.py` (try `rst`, fallback to `md`). I got the description unto pypi successfully (somehow), although not sure how, considering this code. Maybe I forgot to commit the fix somewhere.